### ISSUE TITLE
Sample colors

### ISSUE THREAD

**Issue by mgifford** [reaviz/reaviz#288]
Nice to see these colors - https://reaviz.dev/docs/getting-started/colors

Have they been checked for compliance against WCAG 2.1? Will it work in dark mode? 

Accessibility is really important to us.

**Comment by amcdnl** [comment]
@mgifford - The majority of the color schemes are from ColorBrewer so they should be good for a11y. The axis ticks/labels/etc would likely need to be adjusted ( see https://reaviz.dev/docs/getting-started/setup#configuration ) depending on the background to work but that should be easy.

**Comment by mgifford** [comment]
I'm pretty sure that ColorBrewer is focused in on supporting those with color blindness. I just created an upstream issue though to see if perhaps it has been addressed somewhere.

https://github.com/axismaps/colorbrewer/issues/44

Thanks for your super fast response @amcdnl
